app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -382,6 +383,12 @@ class Meta:
             "total_enrolled_clients",
         ]
 
+    def __init__(self, instance=None, data=None, **kwargs):
+        self.should_call_preview_task = (
```

Comment:
Should this maybe also include a check of the current status to ensure this is a *transition*?

e.g. `instance.status != data.get("status") and data.get("status") == NimbusExperiment.Status.PREVIEW`

Probably not practically possible for this to happen, but seems like this should be made idempotent.

Reply:
Yeah it's not a bad point but I think in general `status` will only be provided in `data` when it's being set, so if the status is something other than preview and `status` is in `data`, that's the case we want, and if for some reason we're setting it to `preview` and it's already `preview`, that's such an edge case it's not worth building around.  Additionally the sync task itself is idempotent so calling it again doesn't hurt anything.